### PR TITLE
Support bw6761

### PR DESCRIPTION
**Pull request by liuxiaobleach**

### CODE REVIEW

File: curves/bw6761/g2_conversions.go [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+	for _, gg2Affine := range elements {
+		var newElement icicle.G2PointAffine
+		G2AffineFromGnarkAffine(&gg2Affine, &newElement)
+
+		newElements = append(newElements, newElement)
+	}
+	return newElements
+}
+
+func BatchConvertFromG2AffineThreads(elements []bw6761.G2Affine, routines int) []icicle.G2PointAffine {
```
Comment (ImmanuelSegol) [inline curves/bw6761/g2_conversions.go:75]: @jeremyfelder did we expose this cuda api?

**Review by ImmanuelSegol — APPROVED** [review]

File: curves/bw6761/g2_conversions.go [hunk 2]
```diff
@@ -0,0 +1,118 @@
+package bw6761
+
+import (
+	"fmt"
+	"github.com/consensys/gnark-crypto/ecc/bw6-761"
+	"github.com/consensys/gnark-crypto/ecc/bw6-761/fp"
+	icicle "github.com/ingonyama-zk/icicle/goicicle/curves/bw6761"
+)
+
+func ToGnarkE2(f *icicle.G2Element) *fp.Element {
```
Comment (jeremyfelder) [inline curves/bw6761/g2_conversions.go:10]: nit: change the function name since the return is not an E2 type

**Review by jeremyfelder — APPROVED** [review]
Small nit but otherwise looks good 👍🏻 🚀

File: curves/bw6761/g2_conversions.go (hunk 1 above)
Comment (jeremyfelder) [reply curves/bw6761/g2_conversions.go:75]: No, we don't do conversions between gnark and icicle types on GPU

Comment (liuxiaobleach) [reply curves/bw6761/g2_conversions.go]: This func is removed now.

**Pull request merged**